recipes/libcugraph/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler_version:     # [linux]
+  - 7                   # [linux64]
+cxx_compiler_version:   # [linux]
+  - 7                   # [linux64]
+channel_targets:
+  - conda-forge dev
```

Comment:
should this be `conda-forge libcugraph_dev` instead of just `conda-forge dev`?

Reply:
Thanks, I think you're right, I just updated it in the latest commit.